tox.ini

```diff
@@ -4,16 +4,22 @@ addopts=--tb=short
 [tox]
 envlist =
        py27-{lint,docs},
-       {py26,py27,py32,py33,py34}-django{15,16},
-       {py27,py32,py33,py34}-django{17,18,master}
+       {py27,py32,py33,py34}-django{17,18},
+       {py27,py34,py35}-django{master}
 
 [testenv]
+basepython =
+            py27: python2.7
+            py32: python3.2
+            py33: python3.3
+            py34: python3.4
+            py35: python3.5
```

Comment:
@carltongibson IMHO it's good enough as it stands. We might reconsider that once 1.9 is out.